google/cloud/spanner_v1/pool.py

```diff
@@ -384,12 +385,13 @@ def get(self, timeout=None):  # pylint: disable=arguments-differ
         if timeout is None:
             timeout = self.default_timeout
 
-        ping_after, session = self._sessions.get(block=True, timeout=timeout)
+        last_used, session = self._sessions.get(block=True, timeout=timeout)
 
-        if _NOW() > ping_after:
-            if not session.exists():
-                session = self._new_session()
-                session.create()
+        if _NOW() > last_used + _SESSION_EXPIRY_DURATION:
```

Comment:
Here, my question is that even if the expire duration (1 hr) has passed, we don't know whether or not the session has expired. We need to verify that the session does not exist before creating a new one. 

In the previous code, it calls `session.exists` to verify if a session exists. 

Can we call `ping()` first? If the session has been deleted, then we create a new session.